File: src/DailyNagger.Server/Observability/ApiRequestHeaders.cs
namespace DailyNagger.Server.Observability;

public static class ApiRequestHeaders
{
    public const string CausalityId = "X-DailyNagger-Causality-Id";
    public const string CausalityKeys = "X-DailyNagger-Causality-Keys";
    public const string RequestId = "X-DailyNagger-Request-Id";
}

File: src/DailyNagger.Mobile/src/api/client/apiRequestHeaders.ts
export const apiRequestHeaders = {
  authorization: "Authorization",
  causalityId: "X-DailyNagger-Causality-Id",
  causalityKeys: "X-DailyNagger-Causality-Keys",
  requestId: "X-DailyNagger-Request-Id",
  sentryTrace: "sentry-trace",
} as const;
